### DISCUSSION TITLE
Date Automations

### DISCUSSION THREAD

**Discussion by markpermann** [NotePlan/plugins#252] [Q&A] [answered]
With Date Automations installed, if I enter a command, say `/time`, I get a popup that begins with
"No templates folder found".

I can select "Cancel command" and then get the timestamp. But that seems to me to be one step more than desirable. 

I would think if you have a template, that's followed; else, the default format is executed as documented in the README file. 

Do I have to create a template file if I just want to use the defaults but also avoid the extra (and confusingly worded) step of selecting "Cancel command"?

**Comment by dwertheimer** [comment, marked as answer]
Mark,

Thank you for pointing out this bug. You are right. This plugin is caught in intermediate migration from the old type of _configuration file to the new configuration system which just released. I believe that @mikeerickson is soon to make these date functions as part of a new global plugins system at which point this plugin will be removed. Or, worst case, he will make some global date config/settings available and I will migrate this plugin to use them. You are correct…it should not need any configuration, and it won't when the new one gets released. Stay tuned. @mikeerickson may be able to give you some timing on that. FYI, @codedungeon, here are the  relevant commands:

https://github.com/NotePlan/plugins/blob/main/dwertheimer.DateAutomations/plugin.json

Best,

David

**Reply by markpermann** [reply]
Thanks David! Wasn't sure it was a bug else I'd have reported it that way :) Appreciate the info.

**Reply by mikeerickson** [reply]
@dwertheimer @markpermann I am squashing my final list of bugs and cleaning up documentation.  Time permitting, it could come as soon as early next week. 

I do need to coordinate with @jgclark on a few things but noting major so we should be in good to go very soon
